_episodes/06-gh-pages.md

```diff
@@ -1,92 +1,49 @@
 ---
 layout: episode
-title: "Deploying a project website or homepage to GitHub Pages"
-teaching: 0
-exercises: 20
+title: Hosting websites/homepages on GitHub Pages
+teaching: 20
+exercises: 0
 questions:
-  - How can we have a good-looking project website without hosting it ourselves?
-objectives:
-  - Create a working example which you can take home and adapt for your project.
-keypoints:
-  - Do not maintain own web servers for project websites or homepages.
+  - How to serve a website/homepage using GitHub
 ---
 
-## Create an example project website and host it on GitHub Pages
+## Hosting websites/homepages on GitHub Pages
 
-In this demo we will create a very simple project website and host it on [GitHub
-Pages](https://pages.github.com/). [GitLab](https://about.gitlab.com/features/pages/) and [Bitbucket](https://confluence.atlassian.com/bitbucket/publishing-a-website-on-bitbucket-cloud-221449776.html) also offer a
-very similar solution. The key is that you can give the hosting to
-professionals so that you can focus on your project.
+You can host your personal homepage or group webpage
+or project website on GitHub using
+[GitHub Pages](https://pages.github.com/).
 
-<img src="{{ site.baseurl }}/img/gh-pages.jpg" width="50%">
+[GitLab](https://about.gitlab.com/features/pages/) and
+[Bitbucket](https://confluence.atlassian.com/bitbucket/publishing-a-website-on-bitbucket-cloud-221449776.html)
+also offer a very similar solution.
 
-It is no problem to use a custom domain instead of `*.github.io`.
+Unless you need user authentication or a sophisticated database behind your website,
+[GitHub Pages](https://pages.github.com/) can be a very nice alternative
+to running your own web servers.
 
 This is how all
 [https://coderefinery.org](https://coderefinery.org)
 material is hosted.
 
 ---
 
-### Context
-
-- We will use a particular framework: [Jekyll](https://jekyllrb.com) which generates HTML from Markdown.
-- [Jekyll](https://jekyllrb.com) is one out of many possible tools (static site generators, more on this later).
-- The point of this exercise is not to focus on Jekyll or Markdown but to demonstrate that
-  **you can serve webpages via GitHub/GitLab/Bitbucket pages**.
-- If we wanted we could serve HTML generated by Sphinx from previous exercise via GitHub pages instead
-  of Read the Docs.
-- You can also write HTML directly if you prefer.
+<img src="{{ site.baseurl }}/img/gh-pages.jpg" width="50%">
 
 ---
 
-> ## Exercice: let us try it out!
->
-> - First import [https://github.com/coderefinery/gh-pages-example](https://github.com/coderefinery/gh-pages-example) and clone the imported repository.
-> ```shell
-> $ git clone https://github.com/user/gh-pages-example.git  # adapt user
-> $ cd gh-pages-example
-> ```
-> - Check that you are indeed on the `gh-pages` branch.
-> ```shell
-> $ git status
-> ```
-> - Make some changes to `index.md` or `about.md` or `team.md`, commit and push.
```

Comment:
I agree with using the upstream pages.github.com reference.  But, can we make it more specific what a possible action should be?  As in, "make a markdown index.md and add a new page team.md".  My worry is that being too open-ended means some people don't know where to start (and those that know, can do what they want anyway).

Reply:
Good suggestion. I will tomorrow go through the official step-by-step just to see what I get precisely and adjust.

Reply:
Done.